.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -4,6 +4,7 @@ about: Create a report to help us improve
 title: ""
 labels: bug
 assignees: ""
+type: 'bub'
```

Comment:
```suggestion
type: 'bug'
```

Is this a typo?

Reply:
No, obviously not. I like to have weird issue types 👦